orchagent/tunneldecaporch.cpp

```diff
@@ -803,3 +920,25 @@ IpAddresses TunnelDecapOrch::getDstIpAddresses(std::string tunnelKey)
 
     return tunnelTable[tunnelKey].dst_ip_addrs;
 }
+
+sai_object_id_t TunnelDecapOrch::getTCToDSCPMap(std::string tunnelKey) const
+{
+    auto iter = tunnelTable.find(tunnelKey);
+    if (iter == tunnelTable.end())
+    {
+        SWSS_LOG_ERROR("Tunnel config not found %s", tunnelKey.c_str());
```

Comment:
Please have it as WARN. It is possible for tunnels not have the mapper and we don't want to raise ERR